chainer/iterators/multiprocess_iterator.py

```diff
@@ -62,13 +62,17 @@ def __init__(self, dataset, batch_size, repeat=True, shuffle=True,
 
         self._finalized = None
 
+        self._previous_epoch_detail = -1.
```

Comment:
I thought it is more straightforward to initialize `_previous_epoch_detail` with `None` directly, rather than setting `-1` and convert it to `None` in its getter method. Is there any intention for this design?